assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/codenvy/che.properties

```diff
@@ -37,6 +37,24 @@ che.workspace.auto_snapshot=true
 # Otherwise create a new workspace.
 che.workspace.auto_restore=true
 
+
+# Workspace threads pool configuration, this pool is used for workspace related
+# operations that require asynchronous execution e.g. starting/stopping/snapshotting
+
+# possible values are 'fixed', 'cached'
+che.workspace.pool.type=fixed
```

Comment:
> If not then why do we need these properties here?

I didn't do anything for overwriting them from che.env. 
I have the opposite question, if i put the properties into this place i probably mean that this is a part of workspace server configuration so WHY do i have to put them somewhere else?
From you question it looks like either property is present in _che.properties_ and _somewhere else_ or it's missing from _che.properties_.

The properties are low-level configuration for workspace threads pool, it allows users to provide more dynamic configuration of starting/stopping/snapshotting q.
So where should i put there properties?